scripts/cmake/vcpkg_configure_make.cmake

```
@@ -726,6 +767,10 @@ function(vcpkg_configure_make)
         vcpkg_list(JOIN tmp " " "${var}")
     endforeach()
 
+    foreach(_toolname IN ITEMS AR AS MT NM RANLIB STRIP OBJDUMP DLLTOOL)
+        z_vcpkg_setup_detected_env(${_toolname} CMAKE_${_toolname})
+    endforeach()

```

```suggestion
    foreach(tool IN ITEMS AR AS MT NM RANLIB STRIP OBJDUMP DLLTOOL)
        z_vcpkg_setup_detected_env("${tool}" "CMAKE_${tool}")
    endforeach()
```

is there a reason this list is different from `cm_TOOLS`?

---

as you can see the main difference are CC CXX and LD.